ksp_plugin/interface.cpp

```diff
@@ -576,6 +577,38 @@ void __cdecl principia__EndInitialization(Plugin* const plugin) {
   return m.Return();
 }
 
+int __cdecl principia__EquipotentialCount(Plugin* const plugin) {
+  journal::Method<journal::EquipotentialCount> m({plugin});
+  CHECK_NOTNULL(plugin);
+  plugin->geometric_potential_plotter().RefreshEquipotentials();
```

Comment:
Why do we refresh if we are not in a rotating pulsating frame?  That seems wasteful.

Reply:
Re**fresh**Equipotentials isn’t Re**quest**Equipotentials (the terminology is the same as in the analyser), the most it does is a swap.